utils/logger.py

```
@@ -2,33 +2,37 @@
 import os
 import datetime
 
+
 def log(message):
```

Ruff (0.11.9)

Missing return type annotation for public function `log`

Add return type annotation: `None`

(ANN201)